packages/modal/src/Modal.story.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import Modal, { ModalSize } from './index';
+
+function Example() {
+  const [active, setActive] = useState(false);
+
+  return (
+    <>
+      <button onClick={() => setActive(active => !active)}>Open Modal</button>
```

Comment:
Why not:
```
<button onClick={() => setActive(!active)}>Open Modal</button>
```

`active` is already available in scope without having to use a callback with `setActive`.

Reply:
👍